packages/compass-components/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "allowJs": false,
+    "esModuleInterop": true,
+    "downlevelIteration": true,
+    "declaration": true,
+    "jsx": "react",
+    "lib": [
+      "dom",
+      "es2018"
+    ],
+    "module": "commonjs",
+    "outDir": "lib",
+    "removeComments": true,
+    "strictNullChecks": false,
```

Comment:
I would also set this to `true`, if we can – it’s certainly helpful to have it enabled in mongosh